frontend/integration-tests/tests/base.scenario.ts

```diff
@@ -27,6 +28,9 @@ describe('Create a test namespace', () => {
         .$('#confirm-action')
         .click();
       await browser.wait(until.urlContains(`/${testName}`), BROWSER_TIMEOUT);
+      execSync(
+        `oc label namespace ${testName} security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=baseline --overwrite=true`,
```

Comment:
Can we update the pod to have the right security values instead?

Reply:
#11894 is an example.

Reply:
ack